### PR TITLE
Add acceptance tests

### PR DESCRIPTION
**Pull request by jgmize**

### CODE REVIEW

**Comment by jgmize** [comment]
To run:

`py.test --acceptance`

This will run against localhost:8000 by default, and if you don't have a recent db restored from prod (or manually created pages with identical slugs) the tests will fail. To run against dev:

`py.test --acceptance --baseurl=https://dev.masterfirefoxos.com`

**Comment by jgmize** [comment]
The commands above assume you are already in a shell with all deps installed, such as "fig run web bash" after a "fig build web" to install the new dependencies.

**Comment by jgmize** [comment]
test this please

**Comment by jgmize** [comment]
test this please

**Comment by jgmize** [comment]
To run the tests against dev in parallel with 7 subprocesses:

`fig run web py.test --acceptance --baseurl=https://dev.masterfirefoxos.com -n 7`

**Comment by jgmize** [comment]
@glogiotatidis r?

File: bin/jenkins/acceptancetests.sh [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+set -ex
+
+COMMIT="${ghprbActualCommit:=$GIT_COMMIT}"
+
+docker run $DOCKER_REPOSITORY:$COMMIT sh -c \
+"DATABASE_URL=sqlite:// DEBUG=False ALLOWED_HOSTS=localhost SECRET_KEY=test \
+PYTHONDONTWRITEBYTECODE=1 py.test --acceptance --baseurl=$BASE_URL -n $NUM_PROCS"
```
Comment (jgmize) [inline bin/jenkins/acceptancetests.sh]: example run of this script: https://ci.masterfirefoxos.com/job/jgmize_acceptance_tests/5/console

File: pytest.ini [hunk 2]
```diff
@@ -0,0 +1,2 @@
+[pytest]
+DJANGO_SETTINGS_MODULE = masterfirefoxos.settings
```
Comment (glogiotatidis) [inline pytest.ini]: Thanks!

**Comment by glogiotatidis** [comment]
This looks great, thanks @jgmize. I did a tiny refactor on the run script to make it easier to read and maintain. Pull requested here #111

**Pull request merged**